ApplicationLibCode/Commands/RicThemeColorEditorFeature.cpp

```diff
@@ -65,9 +65,9 @@ void RicThemeColorEditorFeature::onActionTriggered( bool isChecked )
 
     QComboBox*                          themeSelector = new QComboBox();
     caf::AppEnum<RiaDefines::ThemeEnum> themes;
```

Comment:
themes is now unused?

Reply:
themes works for 3D view, but is broken for some plot types

Reply:
The variable themes is unused?

Reply:
Will fix unused variable